download-bergamot-translator-wasm-artifacts.sh

```diff
@@ -1,8 +1,8 @@
-# The artifacts are built from the same source code as this commit: https://github.com/mozilla/bergamot-translator/commit/48fde5e1985f1d9badebddd526c53f29cf3704a9
-# Links were fetched manually from https://app.circleci.com/pipelines/github/mozilla/bergamot-translator/42/workflows/b46ed52b-b237-4472-976c-11e6a6b90292/jobs/46/artifacts
+# The artifacts are built from the same source code as this commit: https://github.com/mozilla/bergamot-translator/commit/a8cd998720bc34a6ab0d7615557cb402d4326a48
+# Links were fetched manually from https://app.circleci.com/pipelines/github/mozilla/bergamot-translator/64/workflows/6b9c3f45-f900-414d-8060-10cf2e6ec32b/jobs/69/artifacts
```

Comment:
I checked these artifacts and wormhole is enabled there. So, I confirm that the artifacts are sound 👍